docs/agent/changelog.md

```diff
@@ -6,6 +6,18 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.15.0 - \[2024-08-29\]
```

Comment:
@russorat Added your's and warrens comments, any thoughts? Nothing is sacred here, happy to clean up the language further.

*Note*:

- I'll be updating the use of the word "global configurations" to "agent configurations" on this page in the docs (on my _other_ pr).

<img width="408" alt="image" src="https://github.com/user-attachments/assets/6f41bbf5-b077-42c4-9a58-01f353a760d4">